spec/10-overview.md

```diff
@@ -1,16 +1,13 @@
 # Overview
 
-Trace context consists of a set of properties crucial for event correlation in distributed applications. Trace context positions requests in a <a>distributed trace</a>. If you look at a <a>distributed trace<a> as a graph of correlated requests - a single request can be part of multiple graphs. For example, let's say component `A` calls component `B` which in turn calls `C`. One distributed tracing system may see component `A` calling component `B`. Another may see all three components communicating. So the call from `B` to `C` will carry trace context about service `A` as well as information about its direct communication upstream to `B`.
+Trace context is split into two individual propagation fields supporting interoperability and vendor-specifc extensability:

```

Comment:
Was the definition of tracestate as a way to identify request in multiple graphs moved somewhere? I believe it is crutial for understainding the semantics

Reply:
@AloisReitbauer could you comment on that as you did the original change?

Reply:
The original text was super confusing. A lot of concepts like trace graphs etc. People not familiar with the space will have a hard time to understand this. Also explaining multi hop trace text-only with support images makes things hard to grasp. 

I put in here how we solve the problem of interoperability and I believe this is what should be in this section.

I also see the need to have probably an entire section on how to properly use trace-context du build distributed traces. This will be super helpful and I am all in describing this. 

I am fine taking the first round of this section. "Building Distributed traces with Trace-Context"

Reply:
Yes, it will be great to take the first part of it. Without multi-graph positioning explanation it's hard to understand the problem of interoperability and one of the key ideas behind `tracestate`. And examples of `tracestate` with `parent-id` in them looks out of place. As spec didn't explain the concept of using your `tracestate` entry for your own positioning. I agree that it needs better explanation and pictures. But I think simply removing this text makes spec less descriptive.

Reply:
Let us still merge this one and handle the detailed usage in a separate PR. I don't see why this should prevent us from merging this PR